makeRunSet.sh

```diff
@@ -0,0 +1,87 @@
+
+export OMP_NUM_THREADS=20
+
+# Directory to save the runs to
+runsdir='/projects/lanl/Carrington/ModelOutputs/PTM/test'
+
+# Directory to save the fields to
+fieldsdir='/projects/lanl/Carrington/ModelInputs/PTMFields/test'
+
+# Directory for PTM (should probably get this automatically?)
+ptmdir='/home/mengel/github/SHIELDS-PTM'
```

Comment:
Wouldn't this just be
~~~
ptmdir=`pwd`
~~~
?

Reply:
Yes, it would be. I had it placed in a different folder on the scheme which necessitated the ptmdir. If it is just being run from the base directory pwd works.

Reply:
Since the script is in the project base directory, should we just switch to `pwd`? It's one fewer hardcoded path to deal with later...